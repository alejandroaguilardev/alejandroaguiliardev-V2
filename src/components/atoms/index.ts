export * from "./CardContent/CardContent";
export * from "./Copyright/Copyright";
export * from "./Curriculum/Curriculum";
export * from "./ListOption/ListOption";
export * from "./Logotype/Logotype";
export * from "./Hero/Hero";
export * from "./Networks/Networks";
export * from "./PageRoute/PageRoute";
export * from "./TitleSection/TitleSection";
