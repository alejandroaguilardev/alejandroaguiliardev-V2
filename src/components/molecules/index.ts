export * from './CardItem/CardItem'
export * from './Menu/Menu'
export * from './MenuResponsive/MenuResponsive'
export * from './SocialNetworks/SocialNetworks'