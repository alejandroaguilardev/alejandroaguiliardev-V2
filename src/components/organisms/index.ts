export * from "./About/About";
export * from "./ContainerSocialNetworks/ContainerSocialNetworks";
export * from "./Education/Education";
export * from "./Experience/Experience";
export * from "./Footer/Footer";
export * from "./Navbar/Navbar";
export * from "./Projects/Projects";
export * from "./Skills/Skills";
