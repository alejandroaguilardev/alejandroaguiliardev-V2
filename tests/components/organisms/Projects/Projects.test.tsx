import { render, screen } from "@testing-library/react";
import { Projects } from "../../../../src/components/organisms";


describe("Project Test", () => {
	const title = "Project";
	const description = "description Project";

	test("toMatchSnapshot", () => {
		const { container } = render(
			<Projects title={title} description={description} dataProjects={[]} />
		);

		expect(container).toMatchSnapshot();
		expect(screen.getByText(title)).toBeTruthy();
		expect(screen.getByText(description)).toBeTruthy();
	});
});
