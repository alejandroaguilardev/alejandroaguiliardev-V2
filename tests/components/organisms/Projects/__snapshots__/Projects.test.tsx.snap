// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Test toMatchSnapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-1yjvs5a"
    >
      <h2
        class="MuiTypography-root MuiTypography-h2 MuiTypography-alignCenter css-162zxzj-MuiTypography-root"
        role="title"
      >
        Project
      </h2>
      <p
        class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter css-1mqboxc-MuiTypography-root"
        role="description"
      >
        description Project
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5 css-1ld3b9g-MuiGrid-root"
    />
  </div>
</div>
`;
